text/0000-dot-node-version-file-support.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: dot_node_version_file_support
+- Start Date: 2025-03-04
+- RFC PR: (leave this empty)
+- Volta Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Introduce support for a `.node-version` file, enabling it as a fallback source for determining project-specific Node versions. If node version is not specified in `package.json`, fallback to `.node-version` before using global defaults.
+
+# Motivation
+[motivation]: #motivation
+
+The aim is to align Volta with other popular Node version managers like fnm, which support `.node-version`. This enhances flexibility and user experience by allowing developers to specify their preferred Node versions in a widely recognized format.
```

Comment:
2/5 (minor preference, non-blocking) 

Maybe add a few more references (so it's clear it's not just `fnm`), also cross link to the tools websites/repos (so we can poke at them as we review).